### PR TITLE
Enable sitemap.xml

### PR DESCRIPTION
**Pull request by Jolanrensen**
Fixes https://github.com/Kotlin/dataframe/issues/697

enables sitemap according to https://www.jetbrains.com/help/writerside/generate-sitemap.html

### CODE REVIEW

File: docs/StardustDocs/cfg/buildprofiles.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
         <enable-browser-edits>true</enable-browser-edits>
         <browser-edits-url>https://github.com/Kotlin/dataframe/edit/master/docs/StardustDocs/</browser-edits-url>
     </variables>
+    <sitemap priority="0.35" change-frequency="monthly"/>
```
Comment (zaleslaw) [inline docs/StardustDocs/cfg/buildprofiles.xml:20]: When do you define the priority number, how do you generate these numbers?

**Review by zaleslaw — APPROVED** [review]

**Comment by zaleslaw** [comment]
@Jolanrensen thanks for carrying about this question

File: docs/StardustDocs/cfg/buildprofiles.xml (hunk 1 above)
Comment (Jolanrensen) [reply docs/StardustDocs/cfg/buildprofiles.xml:20]: Nope, just what the docs says https://www.jetbrains.com/help/writerside/generate-sitemap.html. To be honest, I'm not sure what it does.

Comment (Jolanrensen) [reply docs/StardustDocs/cfg/buildprofiles.xml:20]: I could probably just leave out the priority, since it's optional

**Comment by Jolanrensen** [comment]
We need to check with WriterSide that our version gets updated so the sitemap generation feature works. It probably won't hurt to merge this already, waiting for the update, but we need to take into account it won't work right away.

**Pull request merged**